Makefile

```diff
@@ -28,7 +28,14 @@ install:
 
 check: pytest doctest
 
-# Run py.test tests. You can environment variable o for pytest options
+# FIXME More directories will be added as Makefiles get improved
+#: Remove derived files
+clean:
+	for dir in mathics/doc; do \
+	   $(MAKE) -C $$dir clean; \
```

Comment:
We should place `"`s around `$$dir`.

Reply:
Fixed on https://github.com/mathics/Mathics/pull/887/commits/c7be29e0337a3a4a86d20d832f706b37c841f4ef.